frontend/react/style-guide.md

```diff
@@ -70,7 +70,7 @@ src
 └───propTypes
 │   | Model1.js
 │   │ Model2.js
```

Comment:
so, if we already have a Typescript section, maybe we could update this section, deleteing propTypes, and change the extension files  from .js to .ts or .tsx